test/src-tests/index-test.js

```diff
@@ -124,12 +186,60 @@ describe('find-parent/index', function() {
       element = document.getElementsByTagName('a')[0];
     });
 
+    it('should throw if opts is truthy and not an object', function() {
+      assert.throws(function() {
+        findParent.byClassName(element, 'nonexistant', 1);
+      }, 'to be an object');
+
+      assert.throws(function() {
+        findParent.byClassName(element, 'nonexistant', []);
+      }, 'to be an object');
+
+      assert.throws(function() {
+        findParent.byClassName(element, 'nonexistant', 'not-empty');
+      }, 'to be an object');
+    });
+
+    it('should throw if opts is truthy and not an object', function() {
```

Comment:
Did you intend for this and the test above it to be in different describe blocks?